packages/os/bootstrap-containers-toml

```diff
@@ -0,0 +1,16 @@
+[required-extensions]
+bootstrap-containers = "v1"
++++
+{{#if settings.bootstrap-containers}}
+{{#each settings.bootstrap-containers}}
+["{{@key}}"]
+source = "{{this.source}}"
+mode = "{{this.mode}}"
+{{#if this.user-data}}
+user-data = "{{this.user-data}}"
+{{/if}}
```

Comment:
User data is opaque so we assume it may contain secrets; it must be rendered to a part of `/etc` with a special label that doesn't allow read access to unprivileged containers.